app/assets/javascripts/file-input-handler.js

```diff
@@ -12,17 +12,21 @@ function toggleSubmitButton(enabled) {
     }
 }
 
-function onFileSelected(id, targetURL, csrf) {
+function onFileSelected(id, targetURL) {
     return function(event) {
         const files = event.target.files;
 
+        const csrfToken = document.querySelector('input[name="csrfToken"]').value;
+
+//        document.write (csrfToken); // Please only use this for manual testing purposes. Uncomment this to test the csrf token or use console and check document.querySelector('input[name="csrfToken"]').value;
+
         if (files.length > 0) {
```

Comment:
Tested this and it works as expected. 

**Form with csrf token**
<img width="1791" alt="Screenshot 2024-05-22 at 10 13 06" src="https://github.com/hmrc/binding-tariff-trader-frontend/assets/63714619/4494f0ff-558a-43af-a5a6-4a8c5a6f2160">

**Console check**

<img width="1791" alt="Screenshot 2024-05-22 at 10 13 27" src="https://github.com/hmrc/binding-tariff-trader-frontend/assets/63714619/1b19ae78-37e5-4b22-80bd-a8e61019045b">

**Document.write check**

<img width="1786" alt="Screenshot 2024-05-22 at 10 13 48" src="https://github.com/hmrc/binding-tariff-trader-frontend/assets/63714619/2f0759c9-c97f-408b-8c81-270fc05915f2">